config/checkstyle/checkstyle.xml

```diff
@@ -3,8 +3,10 @@
 
 <module name="Checker">
     <property name="severity" value="error" />
+    <module name="SuppressWarningsFilter" />
     <module name="TreeWalker">
         <property name="tabWidth" value="4" />
+        <module name="SuppressWarningsHolder" />
```

Comment:
For my edification, why these changes?

Reply:
these changes allow me to use the @SuppressWarnings annotation on one method that has 8 parameters. `UmsUsersStateProvider.handleUser()`